release_notes.md

```diff
@@ -1,34 +1,32 @@
 
-+ **Supporting multi-hot in parquet data reader**: We've added multi-hot support of parquet dataset file. You can train models with a dataset containing both one hot and multi-hot slots. Currently, only iteration-based training mode is supported with parquet dataset format.
++ **DIN Model Support**: 
 
-+ **Supporting Mixed-precision in more layers**: The MultiCross layer now supports mixed-precision (FP16). As a result, all layers now support mixed-precision.
++ **Multi-Hot Support for Parquet Datasets**: We've added multi-hot support for parquet datasets, so you can now train models with a paraquet dataset that contains both one hot and multi-hot slots.
 
-+ **Supporting Optimizer States Save/Load for continued training**: You can choose to store optimizer states updated during the training to files like you save trained weights of your model. For instance, Adam optimizer has the first moment (m) and the second moment (v), which keep being updated across training iterations. By default, they are initialized with zeros, but you can specify a set of optimizer state files to recover their previous values. For more details, find `dense_opt_states_file` and `sparse_opt_states_file` in  [Python Interface](docs/python_interface.md#load_dense_weights-method).
++ **Mixed Precision (FP16) Support in More Layers**: The MultiCross layer now supports mixed precision (FP16). All layers now support FP16.
 
-+ **Supporting FP16 in inference**: We have supported FP16 for the inference pipeline which means that half precision forward propagation can be adopted by the dense layers during inference.
++ **Mixed Precision (FP16) Support in Inference**: We've added FP16 support for the inference pipeline. Therefore, dense layers can now adopt half precision forward propagation during inference.
 
-+ **Embedding cache release in submodule**: We’ve separated the header/source code/document of GPU embedding cache data structure into a stand-alone folder. Now it will be compiled into a stand-alone library file, so your application programs can be directly linked against it just as HugeCTR does.   
++ **Optimizer State Enhancements**: You can choose to store optimizer states that are updated during training as files, such as the Adam optimizer's first moment (m) and second moment (v) that are updated across training iterations. By default, the optimizer states are initialized with zeros, but you can specify a set of optimizer state files to recover their previous values. For more information, see `dense_opt_states_file` and `sparse_opt_states_file` in  [Python Interface](docs/python_interface.md#load_dense_weights-method).
```

Comment:
I think it's better to emphasize continued training here, because **Optimizer State Enhancements** an important feature for it.